src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```
@@ -0,0 +1,63 @@
+approvedScripts=Approved scripts
+
+# approved scripts
+metadataGatheringDisabled=The metadata gathering was disabled. \
+  <p>You can re-enable it permanently by removing the system property: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.metadataGathering</code> </p>\
+  <p>or temporarily (until restart) by running this in the Script Console: <br />\
+  <code>org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApproval.METADATA_GATHERING = true</code></p>
+
+## table headers
+approvedHash=Hash
+approvedHash_tooltip=The hash could be seen as a signature of the script. It's used as an identifier to reference the content of the script.
+approvedExpandCode=Code
+approvedLanguage=Language
+approvedScriptLength=Length
+approvedScriptLength_tooltip=Length of the script in characters
+usageCount=# of uses
+usageCount_tooltip=Usage since the introduction of metadata
```

```suggestion
usageCount_tooltip=Uses since the introduction of metadata
```

`usage` and `use` are distinct things, if `use` applies, it should be used consistently throughout to apply to a given topic. In the rare cases where it doesn't, you can then use `usage`, but again, consistently for a given concept.